### ISSUE TITLE
Deployment builds generate an invalid Go linker command-line

### ISSUE THREAD

**Issue by creachadair** [bblfsh/bblfshd#243]
When Travis builds for deployment, e.g., upon seeing a new release tag, it generates a build stamp with a space in it. This unfortunately breaks when the `go` tool strips off the outside quotes from its `-ldflags` argument, because it results in an extra unrecognized argument.

An example from a recent push build:

```
BBLFSHD_BUILD="2019-02-01T05:51:47 PM+0000"
```

This is not the time format we wanted, which is (for example):

```shell
$ date +%FT%X%z
2019-02-01T10:07:41-0800
```

I'm not yet sure what differs in the deployment config.

There are, I think, two things to address here:

- [x] The date string is coming out differently in the release build than the test build (#246).
- [x] The version string quoting is being lost (#244).

**Comment by creachadair** [comment]
Ok! Progress: The quoting issues are _actually_ fixed now. The CI still [generates a different timestamp](https://travis-ci.org/bblfsh/bblfshd/builds/489161481), e.g. `2019-02-05T05:54:55 PM+0000` instead of `2019-02-05T05:54:55+0700` (note the inserted `PM`).

**Comment by creachadair** [comment]
This is now fixed and deployed in release v2.11.4.
